plugins/modules/terraform.py

```diff
@@ -27,18 +25,16 @@
   state:
     choices: ['planned', 'present', 'absent']
     description:
-      - Goal state of given stage/project
+      - Goal state of given stage/project.
     type: str
     default: present
   binary_path:
     description:
-      - The path of a terraform binary to use, relative to the 'service_path'
-        unless you supply an absolute path.
+      - The path of a terraform binary to use, relative to the 'service_path' unless you supply an absolute path.
```

Comment:
```suggestion
      - The path of a C(terraform) binary to use, relative to the 'service_path' unless you supply an absolute path.
```
I'm wondering whether `service_path` is an option of this module (that has a different name nowadays). But :shrug: